pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.8

```diff
@@ -390,18 +401,34 @@ even if the current NixOS systems uses flakes.
 .Pp
 In addition,
 .Nm
-accepts various Nix-related flags, including
+accepts following builder flags:
```

Comment:
Aren’t most of these *Nix* flags and not specific to the `nixos-rebuild`? I understand documenting commonly used flags, but I fear that this adds to much noise to the documentation and will quickly get out of sync (e.g. it already is, nixos-rebuild also accepts flake lock file flags like `--no-update-lock-file` that are not listed here).

Reply:
I have addressed and added flake-related options in new commit.